plugin/trino-hive/src/main/java/io/trino/plugin/hive/s3select/S3SelectPushdown.java

```diff
@@ -106,9 +106,12 @@ private static boolean isInputFormatSupported(Properties schema)
     public static boolean isCompressionCodecSupported(InputFormat<?, ?> inputFormat, Path path)
     {
         if (inputFormat instanceof TextInputFormat) {
-            return getCompressionCodec((TextInputFormat) inputFormat, path)
+            var compressionCodec = getCompressionCodec((TextInputFormat) inputFormat, path);
```

Comment:
please see https://github.com/trinodb/trino/blob/master/.github/DEVELOPMENT.md#avoid-var

Reply:
Fixed in the latest commit.